Makefile

```diff
@@ -1,7 +1,4 @@
-.PHONY: dev logs stop clean build build-emr_eks_container_example_dag-image build-aws build-google-cloud build-run docs
-.PHONY: restart restart-all run-tests run-static-checks run-mypy run-local-lineage-server test-rc-deps shell help
```

Comment:
should we add targets for run-static-checks & run-mypy?

Reply:
makes sense, since both run-static-checks & run-mypy target does not exist in makefile at the moment so removing it. we should add it for sure in future